de.json

```
@@ -1906,42 +1906,42 @@
-  "log_in_with_qr_step_3": "Select your name in the top left",
-  "feedback_charger_image": "Can you take a photo of the charger?",
-  "continue_on_phone": "Continue on your phone",
-  "general_connection_error_title": "Connection error",
-  "general_connection_error_description": "Please try again and if the issue persists, please contact support.",
-  "store_review_title": "Enjoying ABRP?",
-  "store_review_description": "We would like to know if you like the app. Your feedback will help us to make a difference!",
-  "love_it": "Love it",
-  "needs_improvement": "Needs improvement",
-  "rate_us_on_app_store": "Rate us on the App Store",
-  "rate_us_on_play_store": "Rate us on the Google Play Store",
-  "help_us_improve": "Help us improve",
-  "feedback_improve_description": "Your insights help us grow. Tell us what didn't meet your expectations.",
-  "please_share_your_thoughts": "Please share your thoughts...",
-  "store_review_not_available": "Store review is not available on this device."
-}
\ No newline at end of file
+  "upload_photos": "Fotos hochladen",
+  "take_new_photos": "Neue Fotos aufnehmen",
+  "add_charger_image_detailed_description": "Versuche, das Ladegerät und die Umgebung zu fotografieren. So können andere das Ladegerät finden und wissen, was sie erwartet.",
+  "add_charger_image_anonymization": "Wir werden Kennzeichen und Gesichter automatisch unkenntlich machen.",
+  "speed_camera_category": "Blitzer",
+  "speed_camera_premium_title": "Eine Warnung vor einem bevorstehenden Blitzer anzeigen",
+  "speed_camera_premium_text": "Blitzer auf der Karte anzeigen",
+  "speed_camera_on_screen_alert": "Warnungen auf dem Bildschirm",
+  "speed_camera_on_screen_alert_description": "Anzeige von Warnhinweisen auf dem Bildschirm vor einem Blitzer",
+  "speed_camera_map_markers": "Kartenmarkierungen",
+  "speed_camera_map_markers_description": "Markierungen für Blitzer auf deiner geplanten Route anzeigen",
+  "speed_camera_type_speed": "Geschwindigkeitsblitzer",
+  "speed_camera_type_average": "Überwachung der Durchschnittsgeschwindigkeit",
+  "speed_camera_type_bus_lane": "Überwachung der Busspur",
+  "speed_camera_type_dangerous_zone": "Blitzer-Zone",
+  "speed_camera_type_mobile": "Mobiler Blitzer",
+  "speed_camera_type_red_light": "Rotlichtblitzer",
+  "speed_camera_type_red_light_and_speed": "Geschwindigkeits- und Rotlichtblitzer",
+  "speed_camera_type_regular_mobile_zone": "Mobile Geschwindigkeitsüberwachungszone",
+  "speed_camera_cm_zone_ending_short": "Ende der Durchschnittsgeschwindigkeitsüberwachung",
+  "speed_camera_cm_zone_starting_short": "Durchschnittsgeschwindigkeitsüberwachung",
+  "log_in_with_qr_step_1": "Starte die ABRP-App auf einem Gerät, auf dem du bereits angemeldet bist",
+  "log_in_with_qr_step_2": "Gehe zur Seite mit den Einstellungen",
+  "log_in_with_qr_step_3": "Wähle deinen Namen oben links",
+  "feedback_charger_image": "Könntest du ein Foto von der Ladestation machen?",
```

```suggestion
  "feedback_charger_image": "Kannst du ein Foto von der Ladestation machen?",
```
Könnte beides gehen...

---

Der Konjunktiv 2 wird hier als höfliche Aufforderung verwendet. "Kannst du" wäre die Frage zur faktischen Fähigkeit und wäre hier nicht angebracht, weil wir den Benutzer ja auffordern wollen.

---

Sehe ich genauso wie DIDoS.